### ISSUE TITLE
gets disconnected after few seconds

### ISSUE THREAD

**Issue by medisfar** [bolt/bolt#3298]
An exception has been thrown during the rendering of a template ("You must be logged in to use this.") in "dashboard/_aside.twig" at line 25.

**Comment by bobdenotter** [comment]
Hi, have you made modifications to this section of config.yml? 

```yml
# Bolt allows some modifications to how 'strict' login sessions are. For every option
# that is set to true, it becomes harder for a bad-willing person to spoof your login
# session. However, it also requires you to re-authenticate more often if you change
# location(ip-address) or your browser has frequent upgrades. Only change these if
# you know what you're doing, and you're having issues with the default settings.
# note: if you change any of these, all current users will automatically be logged off.
cookies_use_remoteaddr: true
cookies_use_browseragent: false
cookies_use_httphost: true

# The length of time a user stays 'logged in'. Change to 0 to end the session when the browser
# is closed. The default is 1209600: two weeks in seconds.
cookies_lifetime: 1209600

# Set the session cookie to a specific domain. Leave blank, unless you know what you're doing.
# When set incorrectly, you might not be able to log on at all.
# If you'd like it to be valid for all subdomains of 'www.example.org', set this to '.example.org'.
cookies_domain:

# If your system is using a custom session configuration (such as a redis handler from a
# PHP extension) then we need to disable our session storage handler. In that case, set
# this setting to false. Do not change this, unless you know what you're doing!
# See: http://silex.sensiolabs.org/doc/providers/session.html#custom-session-configurations
session_use_storage_handler: true
```

**Comment by Umbrielsama** [comment]
Got the same issue here, appears since bolt 2.1.3 update - no changes in the above settings.
May be its connected to issue #2948 !?

Fot me it happens when latesteactivity widget is loaded a few seconds later. Additionally next 4-5 login attempts fail immediatelly.

**Comment by Umbrielsama** [comment]
Just used the affected websites with Firefox instead Chrome as usually and the issue disappears - weird.

**Comment by bobdenotter** [comment]
I'm not sure what would cause this, but in most cases it goes away by:

- Clearing PHP's /tmp folder
- Restarting apache/nginx
- clearing your cookies in the browser.

**Comment by Umbrielsama** [comment]
First both are not possible on most Webspace packages. the log just says the session key is invalid/does not match.
It never happens on my localhost, but on all Webserver, when I visit the backend with Webkit based browsers. The log always says the session key is invalid.
I tested your suggestions on my own server, but the problem stays.

**Comment by bobdenotter** [comment]
@Umbrielsama Are you sure it's since the **2.1.3** update? Which version are you on now?

**Comment by Umbrielsama** [comment]
No, in meantime I also updated an **2.1.1** installation to current version and had the same issue. So I think it happens since this or **2.1.0** release. 

Currently running **2.1.6** everywhere.

May be it is anything browser related instead? It is still strange, that it **never** happens to Firefox or IE, but on Chrome and Opera. No clue - still weird!

**Comment by woutertenbosch** [comment]
I have the same issue! After some debugging I know the reason your session key becomes invalid is because the client ip changes. I suspect this might occur on Chrome while using the Data Saver option, or on Opera when using Opera Turbo. Using these data saving proxies causes your IPs to change.

Workaround:
- disable Chrome Data Saver / Opera Turbo - OR -
- using HTTPs (which is not supported for above technologies) - OR -
- disabling cookies_use_remoteaddr (less secure)

A nicer solution would be to not use the IP, but the subnet I guess?

**Comment by Umbrielsama** [comment]
Nice one. Thanks to my low bandwidth I like these data savers. But thank you - I will deactivate them when working with bolt  these days.

**Comment by GawainLynch** [comment]
Found the issue, and it will be addressed in the upcoming work on Bolt 2.3.

**Comment by GawainLynch** [comment]
Tracking this via #2948

**Comment by bobdenotter** [comment]
Fixed by #3564
